src/api/telemetry/TelemetryReporter/report.ts

```diff
@@ -1,11 +1,11 @@
-const initTime = process.hrtime()
-
 import { TelemetryMetaMetrics } from '../../metrics/MetricNames'
 import { SessionManager } from '../../session/SessionManager'
 import { TelemetryLocalStore } from '../TelemetryStore'
 import { PendingTelemetryDataManager } from './PendingTelemetryDataManager'
 import { TelemetryReporter } from './TelemetryReporter'
 
+const initTime = process.hrtime()
```

Comment:
This has to be placed on the beginning of the file

Reply:
There are some rules overrides on .eslintrc that you may want to update the file locations:
https://github.com/vtex/toolbelt/blob/c0897a9056d01e98618b12d3bd2d450397223524/.eslintrc#L33-L47